docs/quick_start.md

```diff
@@ -0,0 +1,195 @@
+
+Once you have the proper plugin installed, use File > Open Files(s)... and select the image file, or simply drag and drop the image into napari. 
+
+For demo purpose, we will use a sample image that comes with napari.
+
+(1) Open napari IPython console<br>
+<img src="img/IPython.png" alt="IPython console" width="30%"/>
+
+(2) Type
+    
+```python
+from skimage import data
+viewer.add_image(data.cell(), name='cell')
+```
+
+```{code-cell} ipython3
+:tags: [hide-cell]
+
+import napari
+from napari.utils import nbscreenshot
+
+viewer = napari.Viewer()
+
+from skimage import data
+viewer.add_image(data.cell(), name='cell')
+nbscreenshot(viewer)
+```
+
+### Image display adjustment
+
+The layer control panel at the upper left of napari viewer allows you adjust contrast, colormap etc. These settings affect the visualization, but do not affect the underlying data.
+
+To change the image display through [API](https://napari.org/api/stable/napari.layers.Image.html), in IPython console, type
+
+```python
+viewer.layers['cell'].colormap = "yellow"
+```
+
+```{code-cell} ipython3
+:tags: [hide-cell]
+
+viewer.layers['cell'].colormap = "yellow"
+nbscreenshot(viewer)
+```
+
+### Manually label the cell
+
+To measure the area of the cell, we can use labels layer and manually "paint" the cell.
+The labels layer allows you to record the segmentation result by assigning background = 0, and assigning each object with an integer.
+
+(1) Add a new labels layer <br> 
+(2) Click on "paint" <br>
+(3) Circle the cell <br>
+(4) Use "fill" bucket to fill it. <br>
+
+![manual_label](./img/manual_label.gif)
+
+Several plugins can perform automatic segmentation that takes image layer as input and generates labels layer as output. <br>
+Try [cellpose-napari](https://www.napari-hub.org/plugins/cellpose-napari) if you have cell images.
+
++++
+
+### Get the cell area measurement
+
+[skimage.measure.regionprops](https://scikit-image.org/docs/dev/api/skimage.measure.html#skimage.measure.regionprops) provides a good set of features that can be extracted from labels, including area measurement.
+
+In IPython console, type
+
+```python
+from skimage.measure import regionprops
+props = regionprops(viewer.layers['Labels'].data)
+print("the cell area is: ", props[0].area)
+```
+
+Alternatively, try [this plugin](https://www.napari-hub.org/plugins/napari-skimage-regionprops) to have the result in a table form.
+
+**Note:** the area reported by regionprops is the number of pixels. Check pixel size and convert the reported number to physical units.
+
+```{code-cell} ipython3
+:tags: [hide-cell]
+
+viewer.close_all()
+```
+
+### Next steps
+- Try opening 3D or higher dimensional images, and switch to 3D view. <br>
+<img src="./images/ndisplay.png" alt="ndisplay" width="30%"/><br> 
+napari provides the flexibility to handle multi-dimensional data, though the increased dimension could add to overall complexity.
```

Comment:
might remove the line "though the increased dimension could add to overall complexity."  (not wrong, just comes off a little odd)